hindsight/modules/heka/elasticsearch/moz_telemetry.lua

```diff
@@ -0,0 +1,133 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
+-- License, v. 2.0. If a copy of the MPL was not distributed with this
+-- file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+--[[
```

Comment:
Ha this is sitting in my working directory (here are the updated docs that work with the generator, please update the comment block)

# Elasticsearch Encoder for Unified Telemetry Messages

## Module Configuration Table

[Common Options](http://mozilla-services.github.io/lua_sandbox/modules/heka/elasticsearch.html#module-configuration-table-common-options)
```lua
-- Array of Heka message field names that should be passed to Elasticsearch.
fields = {"Payload", "Fields[docType]"} -- required
```
## Functions

### encode

Encodes a Heka unified telemetry message into an ElasticSearch Bulk Load JSON
schema.

*Arguments*
- none

*Return*
- JSON (string)